File: Selenium/Activities/Activity13.java
package Example;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import java.util.List;

public class Activity13 {
	
	public static void main(String[] args) {
		  
		   WebDriver driver = new FirefoxDriver();
				    
		   driver.get("https://training-support.net/webelements/tables");
				   
		   System.out.println("Title is: "+driver.getTitle());
		   
		   List<WebElement> rows = driver.findElements(By.xpath("//table/tbody/tr"));
		   System.out.println("Number of rows: " +rows.size());
		   
		   List<WebElement> cols = driver.findElements(By.xpath("//table/thead/tr/th"));
		   System.out.println("Number of columns: " +cols.size());
		   
		   List<WebElement> rows3 = driver.findElements(By.xpath("//table/tbody/tr[3]"));
		   
		   for (WebElement row3:rows3) {
			   
		   System.out.println("3rd row of the table : " +row3.getText());
		   
		   }
		   
		   WebElement cellValue = driver.findElement(By.xpath("//table/tbody/tr[2]/td[2]"));
	       System.out.println("Second row, second column value: " + cellValue.getText());
		   
		   
	driver.quit();
	}

}

File: Selenium/Activities/activity13.py
from selenium import webdriver
from selenium.webdriver.common.by import By

with webdriver.Firefox() as driver:
   
   driver.get("https://training-support.net/webelements/tables")
   
   print("Page title is: ", driver.title)

   rows = driver.find_elements(By.XPATH, "//table/tbody/tr")
   print("Number of rows: ",len(rows))

   cols = driver.find_elements(By.XPATH, "//table/thead/tr/th")
   print("Number of cols: ",len(cols))

   rows3 = driver.find_elements(By.XPATH, "//table/tbody/tr[3]")
   for row3 in rows3:
      print("3rd rows of the table: ",row3.text)
    
   cellValue = driver.find_element(By.XPATH, "//table/tbody/tr[2]/td[2]")
   print("2nd rows & 2nd cols of the table: ", cellValue.text)

driver.quit()